linux-cachyos-server/PKGBUILD

```diff
@@ -416,7 +412,8 @@ prepare() {
         case "$_preempt" in
             full) scripts/config -e PREEMPT_BUILD -d PREEMPT_NONE -d PREEMPT_VOLUNTARY -e PREEMPT -e PREEMPT_COUNT -e PREEMPTION -e PREEMPT_DYNAMIC;;
             voluntary) scripts/config -e PREEMPT_BUILD -d PREEMPT_NONE -e PREEMPT_VOLUNTARY -d PREEMPT -e PREEMPT_COUNT -e PREEMPTION -d PREEMPT_DYNAMIC;;
-            server) scripts/config -e PREEMPT_NONE_BUILD -e PREEMPT_NONE -d PREEMPT_VOLUNTARY -d PREEMPT -d PREEMPTION -d PREEMPT_DYNAMIC;;
+            none) scripts/config -e PREEMPT_NONE_BUILD -e PREEMPT_NONE -d PREEMPT_VOLUNTARY -d PREEMPT -d PREEMPTION -d PREEMPT_DYNAMIC;;
+            lazy) scripts/config -e PREEMPT_BUILD -d PREEMPT_NONE -d PREEMPT_VOLUNTARY -e PREEMPT_LAZY -e PREEMPT_COUNT -e PREEMPTION -e PREEMPT_DYNAMIC;;
```

Comment:
We should disable `PREEMPT_DYNAMIC` for servers running `PREEMPT_LAZY` since theres a choice of `PREEMPT_RCU` being made at compile time (can't change with `preempt=` flag).

Reply:
https://lore.kernel.org/lkml/20241106201758.428310-1-ankur.a.arora@oracle.com/ Relevant discussion thread.

Reply:
Looks like this isn't possible without the above patchset.